examples/server.rs

```diff
@@ -54,7 +54,11 @@ fn main() {
     env_logger::init().unwrap();
 
     let mut chain = Chain::new(hello_world);
-    chain.link_after(HandlebarsEngine::new("./examples/templates/", ".hbs"));
+    let mut hbse = HandlebarsEngine::new();
```

Comment:
How about having a shortcut to do exactly that (so you don't have to break compat)?

* `raw` does what your `new` does.
* `new` calls `raw`, adds one directory source with the given params and reloads.